wpmcpp/src/wpmutils.cpp

```diff
@@ -995,18 +1001,9 @@ QList<HANDLE> WPMUtils::getProcessHandlesLockingDirectory(const QString& dir)
 
     // >= Windows Vista
     if (osvi.dwMajorVersion >= 6) {
-        BOOL WINAPI (*lpfQueryFullProcessImageName)(

```

Comment:
Because QueryFullProcessImageNameW can be accessed directly if the program links kernel32.lib. If it is not working with mingw, I could change it.